src/common/legacy_config_opts.h

```diff
@@ -1524,6 +1524,8 @@ OPTION(rgw_torrent_encoding, OPT_STR)    // torrent field encoding
 OPTION(rgw_torrent_origin, OPT_STR)    // torrent origin
 OPTION(rgw_torrent_sha_unit, OPT_INT)    // torrent field piece length 512K
 
+OPTION(rgw_d3n_l1_local_datacache_plugins, OPT_STR) //choice of l1 local cache
```

Comment:
I think this name could be improved;  don't need l1?; instead of "plugins" how about "backend"--clarifying what the plugin does and making it singular; you can't use more than one at once, can you?

Reply:
Yes.  This also means that use l1 datacache to represent D3N's default local datacache, making the configuration of l1 datacache and IOC cache clearer。
And I have replaced plugins with backend.